.gitignore

```diff
@@ -1,8 +1,9 @@
-*.sublime-project
-*.sublime-workspace
-.DS_Store
-/.jekyll-metadata
-/_site/
-/benchmarks/generated
-/node_modules/
-/npm-debug.log
```

Comment:
No worries, I'm pretty new to this as well :ghost:

I am genuinely confused as to why it flags all the lines as changed even though most of them are visually identical.